### PR TITLE
Add blacken-docs to pre commit 

### PR DESCRIPTION
**Pull request by benjeffery**
[blacken-docs](https://github.com/asottile/blacken-docs) auto formats code in documentation.

### CODE REVIEW

File: python/tskit/trees.py [hunk 1]
```diff
@@ -4703,12 +4704,12 @@ def Tajimas_D(self, sample_sets=None, windows=None, mode="site"):
...
-            h = 1 + 1/2 + ... + 1/(n-1)
-            g = 1 + 1/2^2 + ... + 1/(n-1)^2
-            a = (n+1)/(3*(n-1)*h) - 1/h^2
-            b = 2*(n^2 + n + 3)/(9*n*(n-1)) - (n+2)/(h*n) + g/h^2
-            c = h^2 + g
+            D = (T - S / h) / sqrt(a * S + (b / c) * S * (S - 1))
+            h = 1 + 1 / 2 + ... + 1 / (n - 1)
+            g = 1 + 1 / 2 ^ 2 + ... + 1 / (n - 1) ^ 2
+            a = (n + 1) / (3 * (n - 1) * h) - 1 / h ^ 2
+            b = 2 * (n ^ 2 + n + 3) / (9 * n * (n - 1)) - (n + 2) / (h * n) + g / h ^ 2
```
Comment (benjeffery) [inline python/tskit/trees.py]: I'm not so keen on this, much less readable. Could add `nofmt`?

Comment (jeromekelleher) [reply python/tskit/trees.py]: Looks like it's not Python (h^2 rather than h**2, etc), so no point in being to zealous about black here.

**Comment by jeromekelleher** [comment]
OK, ready to merge after other's have gone through.

File: python/tskit/trees.py (hunk 1 above)
Comment (benjeffery) [reply python/tskit/trees.py]: Removed the `python` specification, fixed.

**Comment by petrelharp** [comment]
lgtm!

**Comment by jeromekelleher** [comment]
OK, just needs a quick rebase to fix conflicts and we're ready to merge!

**Comment by benjeffery** [comment]
Rebased and ready to go!

**Comment by benjeffery** [comment]
Thanks for the merging marathon!!

**Pull request merged**